docs/book/src/topics/managedcluster.md

```diff
@@ -98,17 +98,26 @@ spec:
   location: southcentralus
   resourceGroupName: foo-bar
   sshPublicKey: ${AZURE_SSH_PUBLIC_KEY_B64:=""}
-  subscriptionID: fae7cc14-bfba-4471-9435-f945b42a16dd # fake uuid
+  subscriptionID: 00000000-0000-0000-0000-000000000000 # fake uuid
   version: v1.21.2
   networkPolicy: azure # or calico
   networkPlugin: azure # or kubenet
+  sku: Free # or Paid
+  loadBalancerProfile: # Load balancer profile must specify at most one of ManagedOutboundIPs, OutboundIPPrefixes and OutboundIPs
+    managedOutboundIPs: 2 # 1-100
+    outboundIPPrefixes:
+    - /subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/foo-bar/providers/Microsoft.Network/publicIPPrefixes/my-public-ip-prefix # fake public ip prefix
+    outboundIPs:
+    - /subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/foo-bar/providers/Microsoft.Network/publicIPAddresses/my-public-ip # fake public ip
+    allocatedOutboundPorts: 100 # 0-64000
+    idleTimeoutInMinutes: 10 # 4-120
```

Comment:
Because these fields are optional can we put this in a separate section just like mutitenancy and aad profile. 
The reason why I am saying this is, not all users might want to specify a separate load balancer

Also when you add in the section please link the original loadBalancerProfile docs and if possible a small brief about it.